QModManager/VersionCheck.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Harmony;
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace QModManager
+{
+    internal static class VersionCheck
+    {
+        internal const string nexusmodsURL = "https://nexusmods.com/subnautica/mods/16";
+
+        internal static void Parse(string versionStr)
+        {
+            if (versionStr == null)
+            {
+                UnityEngine.Debug.Log("Could not get latest version!");
+                return;
+            }
+            Version version = new Version(versionStr);
+            if (version == null)
+            {
+                UnityEngine.Debug.Log("Could not get latest version!");
+                return;
+            }
+            if (!version.Equals(QMod.QModManagerVersion) && QModPatcher.erroredMods.Count <= 0) Dialog.Show($"There is a newer version of QModManager available: {version.ToString()}  (current version: {QMod.QModManagerVersion.ToString()})", 
```

Comment:
Again, this is pretty hard to read, even with full width available.  
Please keep single statement code blocks on their own line and nested under their parent if-block.

Reply:
Addressed in commit 35f3801